### PR TITLE
experiment with fixutes for some specs

### PR DESCRIPTION
**Pull request by ericcf**
This uses transactional fixtures with the non-feature specs, and database cleaner with just the feature specs. It decreases the Travis build time by 40-50%.

### CODE REVIEW

File: spec/requests/application_requests_spec.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-require "rails_helper"
+require "rails_fixture_helper"
```
Comment (michaelwehrley) [inline spec/requests/application_requests_spec.rb]: Thanks for sharing these `request` specs.

File: spec/rails_fixture_helper.rb [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+  config.after :suite do
+    Timecop.return
+  end
+
+  # If true, the base class of anonymous controllers will be inferred
+  # automatically. This will be the default behavior in future versions of
+  # rspec-rails.
+  config.infer_base_class_for_anonymous_controllers = false
+
+  config.include AuthenticationHelpers, type: :feature
```
Comment (michaelwehrley) [inline spec/rails_fixture_helper.rb]: Do you need these in here if this is for fixtures?

**Comment by eschlange** [comment]
This looks great, I give the OK to merge. I'll do some local test runs as well and compare them against my changes.

File: spec/rails_fixture_helper.rb (hunk 2 above)
Comment (ericcf) [inline spec/rails_fixture_helper.rb]: Nope! I'll remove.

**Pull request merged**